__tests__/integration/send-tx.test.js

```diff
@@ -1301,26 +1307,6 @@ describe('send tx (custom tokens)', () => {
     expect(changeHtr.total_amount_available).toBe(txOutputSummary.htr.change);
     expect(changeTkb.total_amount_available).toBe(txOutputSummary.tkb.change);
 
-    // Validating that the change addresses are not the same
```

Comment:
Why did you remove this validation?

Reply:
The wallet-lib does not mark the address as used when it is chosen as change, so they will not be different.